likeminds-feed-android-core/src/main/java/com/likeminds/feed/android/core/post/detail/view/LMFeedPostDetailFragment.kt

```diff
@@ -0,0 +1,1403 @@
+        customizePostDetailHeaderView(binding.headerViewPostDetail)
+        customizeCommentComposer(binding.commentComposer)
+        return binding.root
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        receiveExtras()
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initUI()
+        fetchData()
+        initCommentEditText()
+        initListeners()
+        observeData()
+    }
+
+    override fun onResume() {
+        super.onResume()
+        binding.rvPostDetails.initiateAutoPlayer()
+    }
+
+    override fun onPause() {
+        super.onPause()
+        binding.rvPostDetails.destroyAutoPlayer()
+    }
+
+    protected open fun customizePostDetailHeaderView(headerViewPostDetail: LMFeedHeaderView) {
+        headerViewPostDetail.apply {
+            setStyle(LMFeedStyleTransformer.postDetailFragmentViewStyle.headerViewStyle)
+
+            setTitleText(getString(R.string.lm_feed_feed))
+        }
+    }
+
+    protected open fun customizeCommentComposer(commentComposer: LMFeedCommentComposerView) {
+        binding.commentComposer.setCommentInputBoxHint(getString(R.string.lm_feed_write_a_comment))
+        commentComposer.setStyle(LMFeedStyleTransformer.postDetailFragmentViewStyle.commentComposerStyle)
+    }
+
+    private fun receiveExtras() {
+        postDetailExtras = LMFeedExtrasUtil.getParcelable(
+            arguments,
+            LM_FEED_POST_DETAIL_EXTRAS,
+            LMFeedPostDetailExtras::class.java
+        ) ?: throw emptyExtrasException(TAG)
+    }
+
+    private fun initUI() {
+        initPostDetailRecyclerView()
+        //todo:
+//        initMemberTaggingView()
+        initSwipeRefreshLayout()
+    }
+
+    private fun initPostDetailRecyclerView() {
+        fetchPostData()
+        binding.rvPostDetails.apply {
+            setAdapter(
+                this@LMFeedPostDetailFragment,
+                this@LMFeedPostDetailFragment,
+                this@LMFeedPostDetailFragment
+            )
+
+            //set scroll listener
+            val paginationScrollListener =
+                object : LMFeedEndlessRecyclerViewScrollListener(linearLayoutManager) {
+                    override fun onLoadMore(currentPage: Int) {
+                        if (currentPage > 0) {
+                            Log.d("PUI", "load more is called $currentPage")
+                            postDetailViewModel.getPost(postDetailExtras.postId, currentPage)
+                        }
+                    }
+                }
+            setPaginationScrollListener(paginationScrollListener)
+        }
+    }
+
+    private fun initSwipeRefreshLayout() {
+        mSwipeRefreshLayout = binding.swipeRefreshLayout
+        mSwipeRefreshLayout.apply {
+            setColorSchemeColors(
+                //todo: change this color as per the style
```

Comment:
please confirm whether we have added this or not.

Reply:
will be added in further PRs